symbols.js

```diff
@@ -480,6 +559,74 @@ function renderSymbols(searchTerm) {
     }
 }
 
+function handleDragStart(e) {
+    e.target.classList.add('drag');
+	document.getElementsByClassName("symbols")[0].dataset.dragIndex = 
+		Array.from(e.target.parentElement.children).indexOf(e.target);
+}
+
+
+function handleDragEnd(e) {
+	console.log("End", e.target.title)
+    e.target.classList.remove('drag');
+	Array.from(e.target.parentElement.children).forEach(elem => elem.classList.remove("over"));
+}
+
+
+function handleDragOver(e) {
+    e.preventDefault();
+    return false;
+}
+
+function handleDragEnter(e) {
+	Array.from(document.getElementsByClassName("over")).forEach(elem => elem.classList.remove("over"));
+	let target = e.target;
+	while (target) {
+		if (target.classList?.contains("symbol")) {
+			target.classList.add("over");
+			break;
+		}
+		target = target.parentElement
+	}
+}
+
+function handleDragLeave(e) {
+	//console.log("handleDragLeave()", e.target);
+    //e.target.classList.remove('over');
+}
+
+function handleDrop(e) {
+	console.log("drop", e.target.title, e.target)
+	let target = e.target;
+	while (target) {
+		if (target.classList?.contains("symbol")) {
+			target.parentElement.dataset.dragTarget = 
+				Array.from(target.parentElement.children).indexOf(target);
+			
+			symbols.splice(target.parentElement.dataset.dragTarget, 0, symbols.splice(target.parentElement.dataset.dragIndex, 1)[0]);
+			window.localStorage.setItem("symbols", JSON.stringify(symbols));
+			
+			if (parseInt(target.parentElement.dataset.dragIndex) < parseInt(target.parentElement.dataset.dragTarget)) {
+				if (target.nextElementSibling) { 
+					target.nextElementSibling.before(target.parentElement.children[target.parentElement.dataset.dragIndex]);
+				} else {
+					target.parentElement.appendChild(target.parentElement.children[target.parentElement.dataset.dragIndex]);
+				}
+			} else {
+				target.before(target.parentElement.children[target.parentElement.dataset.dragIndex]);
+			}
+			break;
+		}
+		target = target.parentElement;
+	}
+	
+
+	;
```

Comment:
👀

Reply:
Gone in 914415c9dd6387c7983bcf6ddd23508b77b5c2fe ;-)